docs/authentication/configuration/sign-up-modes.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: Sign-up modes
+Description: Learn how to configure and manage user access through sign-up modes in your application.
+---
+
+Clerk supports multiple sign-up modes, giving you flexibility in managing user access to your application.
+
+To configure sign-up modes, navigate to **Configure → User & Authentication → Sign-up Modes**, where you can choose between the available options.
+
+- **[Public](#public)**: Open to all users, allowing anyone to sign up.
+- **[Restricted](#restricted)**: Access is controlled by invitations or manual user creation.
+
+## Public
+
+In Public mode, the sign-up process is open to anyone. This mode is ideal for applications that want broad user access.
+
+## Restricted
+
+In Restricted mode, the user access is controlled by the application admin(s). Users can be added to the application through [invitations](/docs/users/invitations), [manual creation](/docs/users/creating-users), or the [Backend API](/docs/reference/backend-api/tag/Users#operation/CreateUser). This mode is ideal for applications that are in private beta or internal tools.
+
+Additional features available in Restricted mode:
+
+- The `<SignIn />` component will keep the prompt to sign up hidden by default. This is to avoid confusion for users who don't have access.
+
+- The `<SignUp />` is accessible only to users who have been invited and have a valid invitation link. Users who don't have access will see a message indicating that they need an invitation to sign up.
```

Comment:
```suggestion
In **Restricted** mode, user access is controlled by the application admin(s). Users can be added to the application through [invitations](/docs/users/invitations) or [manual user creation](/docs/users/create-users). This mode is ideal for applications that are in private beta or internal tools.

Additional features available in **Restricted** mode:

- The [`<SignIn />`](/docs/components/authentication/sign-in) component will keep the prompt to sign up hidden by default. This is to avoid confusion for users who don't have access.

- The [`<SignUp />`](/docs/components/authentication/sign-up) is accessible only to users who have been invited and have a valid invitation link. Users who don't have access will see a message indicating that they need an invitation to sign up.
```